### PR TITLE
New ioctl command to help iotex-studio link local files

### PR DESCRIPTION
**Pull request by RuneAkko**
Use golang to implement remixd tool.
#2249 
#2247

### CODE REVIEW

File: ioctl/cmd/contract/contractshare.go [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+	if !filepath.IsAbs(absPath) {
+		return output.NewError(output.InputError, "inputed path isn't absolute", nil)
+	}
+
+	info, err := os.Stat(absPath)
+	if err != nil {
+		return output.NewError(output.ReadFileError, "failed to get directory", nil)
+	}
+	if !info.IsDir() {
+		return output.NewError(output.InputError, "input file rather than directory", nil)
```
Comment (raullenchai) [inline ioctl/cmd/contract/contractshare.go:90]: Can we support map dir to IDE?

File: ioctl/cmd/contract/contractshare.go [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+	http.HandleFunc("/", shareFiles)
+	log.Fatal(http.ListenAndServe(*addr, nil))
+
+	return nil
+
+}
+
+func shareFiles(w http.ResponseWriter, r *http.Request) {
+
```
Comment (raullenchai) [inline ioctl/cmd/contract/contractshare.go:117]: remove

**Comment by uu-z** [comment]
For better improvements, we may need to support listening files for hot updates.
ref:https://github.com/ethereum/remixd/blob/5c74bc345192270d1f296cd3629db5a34328fd05/src/services/remixdClient.ts#L198-L230

File: ioctl/cmd/contract/contractshare.go (hunk 1 above)
Comment (RuneAkko) [reply ioctl/cmd/contract/contractshare.go:90]: sure

**Comment by RuneAkko** [comment]
> For better improvements, we may need to support listening files for hot updates.
> ref:https://github.com/ethereum/remixd/blob/5c74bc345192270d1f296cd3629db5a34328fd05/src/services/remixdClient.ts#L198-L230

I used remixd tool link iotex IDE, but  it seems the IDE server couldn't support this feature？

**Pull request merged**